app/components/progress/student-progress-dashboard.tsx

```
@@ -46,9 +46,9 @@ export function StudentProgressDashboard({ studentId }: { studentId: string }) {
 
   useEffect(() => {
     loadProgressData();
-  }, [studentId, selectedTimeRange]);
+  }, [loadProgressData]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)